src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+import { Person } from '../../types';
+import { PersonInTable } from '../PersonInTable';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: FC<Props> = ({ people }) => (
+  <table
+    data-cy="peopleTable"
+    className="table is-striped is-hoverable is-narrow is-fullwidth"
+  >
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Sex</th>
+        <th>Born</th>
+        <th>Died</th>
+        <th>Mother</th>
+        <th>Father</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      {people.map(person => (
+        <PersonInTable person={person} />
```

Comment:
Check the console output for your app:
![image](https://user-images.githubusercontent.com/23013525/236615226-ed3edd44-02f5-4fd4-8250-9ea37f01deb0.png)
You should have your key here, not somewhere down the component chain. Check this article about them https://react.dev/learn/rendering-lists

Reply:
Fixed) Thank you for article)